tests/unit/modules/network/ios/test_ios_snmp_server.py

```diff
@@ -1684,6 +1684,7 @@ def test_ios_snmp_server_gathered(self):
             snmp-server host 172.16.2.99 checktrap  isis hsrp
             snmp-server host 172.16.2.1 version 3 priv newtera  rsrb pim rsvp slb pki
             snmp-server host 172.16.2.1 version 3 noauth replace-User!  slb pki
+            snmp-server host 172.16.2.1 vrf vrf1 version 3 priv newtera1  rsrb pim
```

Comment:
I see there is one extra space that is present when the command is rendered, can you please fix it?